source/industry-solutions/audio-based-AI-diagnostics.txt

```diff
@@ -0,0 +1,246 @@
+MongoDB Atlas Vector Search. At this step, the Atlas Vector Search algorithm
+searches for what is the most similar sound from the ones we previously recorded
+in our audio preparation phase. The result is given back with a percentage of
+similarity and it enables a very accurate diagnosis of the current status of the
+operation of the wind turbine.
+
+These steps are performed repeatedly every second, leveraging fast embedding of
+vector data and quick searches, allowing for real-time monitoring based on
+sound.
+
+.. figure:: /includes/images/industry-solutions/Wind-Turbine-Diagnosis-Demo.svg
+   :figwidth: 1200px
+   :alt: DReal-time wind turbine diagnosis 
+   
+Real-time wind turbine diagnosis by analyzing the emitted audio, allowing us to
+diagnose its status — whether it's operating normally, stopped, or experiencing
+any issues.
+
+Data model approach
+-------------------
+
+The data model for the vectorized audio collection is simple, we have our audio
+preparation collection called “sounds,” which contains documents including a
+status and the url of the related GIF to be displayed on the UI. Once the
+reference audio for each status is vectorized, the embedding is added to the
+document as an array alongside existing fields.
+
+.. figure:: /includes/images/industry-solutions/wind-diagnosis.svg
+   :figwidth: 1200px
+   :alt: Screenshot of documents including a status and the url of the related GIF to be displayed on the UI.
+
+On the reading end, the one-second audio clips we record in real-time are
+vectorized and sent to MongoDB Atlas Vector Search to be compared against
+embeddings from the “sounds” collection above.
+
+Building the solution
+---------------------
+
+**Step 1:** `MongoDB Atlas Connection
+<https://github.com/mongodb-industry-solutions/Wind-Turbine-Diagnostics/blob/main/README.md#1-mongodb-atlas-connection>`__
+
+Create a file called .env in the main directory alongside the add_audio.py file
+and add your atlas connection string in the following format:
+
+MONGO_CONNECTIO_STRING="mongodb+srv://connectionstringfromatlas"
+
+Then copy this file into the nodeUI directory.
+
+**Step 2:** `Install Python Modules <https://github.com/mongodb-industry-solutions/Wind-Turbine-Diagnostics/blob/main/README.md#2-install-python-modules>`__
+
+Install the required python modules listed in the Github repo.
+
+**Step 3:** `Record Audio Files <https://github.com/mongodb-industry-solutions/Wind-Turbine-Diagnostics/blob/main/README.md#3-record-audio-files>`__
+
+1. Run python3 add_audio.py
+2. Select the audio input by typing the relevant number and then press enter. Record each sound in sequence.
+
+**Step 4:** `Create a Search Index <https://github.com/mongodb-industry-solutions/Wind-Turbine-Diagnostics/blob/main/README.md#4-create-a-search-index>`__
+
+Go to MongoDB Atlas and create an Atlas Search Index in the audio database
+sounds collection using the content of searchindex.json.
+
+.. code-block:: sh
+
+   {
+    "mappings": {
```

Comment:
Can we double check the indentation here? It looks like "mappings" is only indented one space whereas the other fields use two spaces for indentation.